### ISSUE TITLE
Android: Work with OpenKeyChain

### ISSUE THREAD

**Issue by kgraefe** [Fmstrat/ownnote#175]
Their are some notes I don't even trust my own server with. I'd like to store them encrypted with PGP. However, the Android app doesn't work very well with OpenKeyChain.

With Google Notes I can write my note in OpenKeyChain, encrypt it and paste it into Google Notes. To decrypt it, I use the "Send to" button to open the note in OpenKeyChain and decrypt it.

With ownnote I there two issues:
* The main issue is that when pasting the encrypted text into to ownNote it starts a new paragraph (adding an additional linefeed) after each line. Those have to be deleted manually before decrypting...
* Another issue is the missing "Send to" menu which makes it inconvenient to send the cipher text to OpenKeyChain.

Is the Android app open source? Maybe I can work on the issue on my own.

**Comment by blaskovic** [comment]
Similar here with 8.0.5.
I can enable it, but I can see just blank page and I am logged out of owncloud.

I think, developer is no longer maintaining this, which is pity..

**Comment by Fmstrat** [comment]
I have on the list supported ownCloud server encryption. Would this work? Or would support for OpenKeyChain still be something you would like?

**Comment by kgraefe** [comment]
Server encryption is useless if you don't trust the server. ;)

**Comment by Fmstrat** [comment]
Good point. Though I thought the point of running an ownCloud server was so that you could trust the server. :)

**Comment by Fmstrat** [comment]
Dependent on: https://github.com/Fmstrat/ownnote/issues/55
